po/fr.po

```diff
@@ -641,12 +639,10 @@ msgstr "Aucun fil sélectionné !"
 #. / "Sort by (f)irsttag/..." and "Reverse Sort by
 #. / (f)irsttag/..." messages
 #: src/feedlistformaction.cpp:135 src/feedlistformaction.cpp:175
-#, fuzzy
 msgid "ftauln"
 msgstr "ptnia"
```

Comment:
This message is missing a translation for `l`, which corresponds to `lastupdated` from the two below messages (which also lack the translation for it.

Reply:
It looks like this hasn't been addressed yet, and it's getting buried in between other comments now, so: :bell: please take a look :wave: :)

Reply:
Oooh it was getting buried indeed, my bad, and thanks for the weave !